src/api/app/policies/notification_policy.rb

```diff
@@ -1,4 +1,16 @@
 class NotificationPolicy < ApplicationPolicy
+  def initialize(user, record, opts = {})
+    super(user, record, opts.merge(ensure_logged_in: true))
+  end
+
+  def index?
+    true
+  end
+
+  def update?
+    true
+  end
```

Comment:
I'm not totally convinced of this approach thus I'd like to further discuss it: the idea is that since we introduced `verify_policy_scoped` in the controller checking for the `subscriber` in `NotificationPolicy#update` it's redundant and also leads to a slowdown of the related endpoint (trade-off), moreover we also have tests in the controller to ensure we cannot update other user notifications. On the other hand if we look at the `NotificationPolicy#update` per-se we don't see that constraint and therefore is misleading.